joomla/joomla-cms
# [4.0] save/cancel button icon color

Pull Request for Issue #29989 

### Summary of Changes
Changes icons to white on save/cancel buttons.

### Testing Instructions
apply pr
run npm ci or download prebuilt package.
go to administrator/index.php?option=com_content&view=article&layout=edit&id=2
click on "images & links"
verify save/cancel icons are white
install "blog sample data"
log into front end as super admin.
go to index.php?option=com_config&view=templates&Itemid=112
verify save/cancel buttons have white icons

### Actual result BEFORE applying this Pull Request
![image](https://user-images.githubusercontent.com/1850089/89331359-3fda6e80-d657-11ea-9d53-2d5573bcec78.png)
![image](https://user-images.githubusercontent.com/1850089/89331373-47017c80-d657-11ea-9825-a487ba3c46f9.png)

### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/1850089/89331211-fb4ed300-d656-11ea-9d60-0728e204f30c.png)

![image](https://user-images.githubusercontent.com/1850089/89331188-f25e0180-d656-11ea-8ba5-58441bab9f28.png)


### Documentation Changes Required
none
